### PR TITLE
Update license file for v0.9.1.

### PR DESCRIPTION
**Pull request by srujann**

### CODE REVIEW

**Comment by akodali18** [comment]
Lgtm

File: open_source_licenses.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 open_source_licenses.txt
 
-Wavefront by VMware Internal Reporter for Java 0.9.0 GA
+Wavefront Internal Reporter for Java 0.9.1 GA
```
Comment (akodali18) [inline open_source_licenses.txt]: Should this be: "Wavefront by VMware" Internal Reporter for Java ?

Comment (srujann) [reply open_source_licenses.txt]: fixed.

**Comment by spramanik4667** [comment]
@srujann Should we update the name in pom.xml as well.

**Pull request merged**